specutils/tests/test_slicing.py

```diff
@@ -15,7 +12,7 @@ def test_spectral_axes():
     sliced_spec1 = spec1[0]
 
     assert isinstance(sliced_spec1, Spectrum1D)
-    assert sliced_spec1.wcs == spec1.wcs
+    #assert sliced_spec1.wcs == spec1.wcs
```

Comment:
Probably should just delete these instead of commenting?

Reply:
👍